bin/metadata.js

```diff
@@ -86,8 +107,16 @@ main(PR_ID, OWNER, REPO).catch((err) => {
 
 function parsePRId(id) {
   // Fast path: numeric string
-  if (`${+id}` === id) { return +id; }
-  const match = id.match(/^https:.*\/pull\/([0-9]+)(?:\/(?:files)?)?$/);
-  if (match !== null) { return +match[1]; }
+  if (!isNaN(id)) {
+    OWNER = argv._[1] || argv.o || 'nodejs';
+    REPO = argv._[2] || argv.r || 'node';
```

Comment:
This would be in `argv._[0]` actually (if supplied)

Reply:
Like previous comment!